app/pages/select/routes.test.js

```diff
@@ -0,0 +1,46 @@
+import request from 'supertest';
+import {
+  testAuthorisedGetPathForUnauthenticatedUser,
+  testAuthorisedGetPathForUnauthorisedUser,
+} from 'buying-catalogue-library';
+import {
+  mockUnauthorisedCookie,
+  mockAuthorisedCookie,
+  setUpFakeApp,
+} from '../../test-utils/routesTestHelper';
+import * as controller from './controller';
+
+jest.mock('../../helpers/api/oapi/getRelatedOrganisations');
+
+describe('GET /organisation/select', () => {
+  const path = '/organisation/select';
+
```

Comment:
Probably worth adding an `afterEach` to reset all mocks – there may not be any issues now but it helps reduce the likelihood of any issues occurring with incorrect mocking in the future.

```js
afterEach(() => {
    jest.resetAllMocks();
  });
```